WooCommerce/Classes/ViewRelated/Dashboard/Blaze/BlazeCampaignDashboardView.swift

```diff
@@ -336,6 +350,37 @@ private struct ProductInfoView: View {
     }
 }
 
+private extension ProductSelectorView.Configuration {
+    static func selectProductForBlaze() -> ProductSelectorView.Configuration {
+        ProductSelectorView.Configuration(
+            searchHeaderBackgroundColor: .listBackground,
+            prefersLargeTitle: false,
+            title: Localization.productSelectorTitle,
+            cancelButtonTitle: Localization.productSelectorCancel,
+            productRowAccessibilityHint: Localization.productRowAccessibilityHint,
+            variableProductRowAccessibilityHint: Localization.variableProductRowAccessibilityHint)
+    }
```

Comment:
Nit: we can create a variable instead of a function here as we don't need any input:

```suggestion
    static let configurationForBlaze = ProductSelectorView.Configuration(
            searchHeaderBackgroundColor: .listBackground,
            prefersLargeTitle: false,
            title: Localization.productSelectorTitle,
            cancelButtonTitle: Localization.productSelectorCancel,
            productRowAccessibilityHint: Localization.productRowAccessibilityHint,
            variableProductRowAccessibilityHint: Localization.variableProductRowAccessibilityHint
        )
```